nntrainer/layers/permute_layer.cpp

```diff
@@ -20,9 +22,57 @@
 
 namespace nntrainer {
 
+bool props::Direction::isValid(const unsigned int &value) const {
```

Comment:
is it direction or axis?? which one is better?

Reply:
Thanks for the suggestion. Changed to `PermuteDims` referring from keras properties :)